pa_matchbatch.sh

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env bash
+set -e
+unset CDPATH
+
+# where to find the agent
+puppet_agent_repo="git@github.com:puppetlabs/puppet-agent.git"
+PUPPET_AGENT_URL=${PUPPET_AGENT_URL:-${puppet_agent_repo}}
+
+# where to find the ticketmatch.rb script
+TICKETMATCH_PATH=${TICKETMATCH_PATH:-$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )}
+
+# where origin actually is
+ORIGIN_PATH=${ORIGIN_PATH:-origin}
```

Comment:
I would rename this to `FETCH_REMOTE`, `ORIGIN_PATH` is a bit weird b/c the clone URL will almost always be the `origin` remote.